src/librustc_session/session.rs

```diff
@@ -529,6 +529,9 @@ impl Session {
     pub fn verbose(&self) -> bool {
         self.opts.debugging_opts.verbose
     }
+    pub fn span_debug(&self) -> bool {
+        self.opts.debugging_opts.span_debug
+    }
```

Comment:
Not sure why these trivial getters exist, all the options are public.
I'd rather remove the existing trivial getters and not introduce new ones.